### ISSUE TITLE
NAV-GAC-Factura venta tipo C COM/02934 - C/20200024691

### ISSUE THREAD

**Issue by bpesquero** [PKFEric/webProxima#1015]
Al hacer la factura de venta de referencia se factura el Término de Potencia, cuando la factura inicial de compra no lleva término de potencia.

**Comment by jorgereal** [comment]
Es un contrato con modo facturación de potencia con maxímetro.

En el algoritmo del cálculo de potencia con maxímetro se contempla que si la potencia del cierre es 0 los cálculos se realicen con la potencia de los precios de contrato.
De ahí que en la factura C/20200024691 haya insertado las líneas de potencia para los 3 periodos con cantidades 10-10-16, como las de los precios de contrato.

¿Añadimos una condición para que en las tipo C con modo facturación de potencia con maxímetro si la potencia del peaje es 0 no inserte término de potencia?

**Comment by bpesquero** [comment]
Exacto. En las tipo C recuerda que lo que hacemos basicamente es replicar lo que viene en la factura de venta

**Comment by jorgereal** [comment]
Hemos añadido el control en el cálculo del término de potencia con maxímetro (en el de sin maxímetro ya estaba) para las facturas C.

Hemos forzado facturación para el contrato COM/02934 fechas 03/09/20 - 28/10/20 y se ha generado el preabono de venta ABP/202001068 y la prefactura de venta FV/2020026908, esta vez sin término de potencia.

Subido a PRE y PRO.
